### ISSUE TITLE
[feature] tensorboard/jupyter notebook integration

### ISSUE THREAD

**Issue by ChristianSch** [ChristianSch/Pythia#14]
- it was proposed to me to include tensorboard somehow
  - either try to read the files created by `SummaryWriter` (not sure if and how this works)
  - or as plain uri
- jupyter notebook integration would be splendid
  - either as html directly via nbconvert
  - or as plain uri

thus, hereby I propose introducing a hypertext resource attribute of models.

**Comment by steel3d** [comment]
Yes, please! This is quite essential - to get tensorboard functionality on something like Azure Notebooks.
